### PR TITLE
set __assetMapFilename__ to false if there is no assetFileName

### PR DESCRIPTION
**Pull request by andreasschacht**
...to match this condition:
https://github.com/RuslanZavacky/ember-cli-ifa/blob/master/addon/initializers/asset-map.js#L11

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -38,17 +38,19 @@ module.exports = {
       fingerprintPrepend = this.app.options.fingerprint.prepend;
     }
 
+    let assetMapContent = 'false';
```
Comment (RuslanZavacky) [inline index.js]: It might be better to set it as `let assetMapContent = null;`

Comment (andreasschacht) [inline index.js]: okay, replaced it

**Comment by RuslanZavacky** [comment]
Thank you @andreasschacht , that's very nice catch! :)

**Comment by andreasschacht** [comment]
@RuslanZavacky You're very welcome, btw. nice addon!

**Pull request merged**